cinema/services.py

```diff
@@ -0,0 +1,5 @@
+def params_to_int(query_string):
+    try:
+        return [int(el) for el in query_string.split(",")]
+    except ValueError:
+        return
```

Comment:
The function should return a more meaningful value or raise an exception when a `ValueError` occurs. Currently, it returns `None`, which might not be the expected behavior. Consider returning an empty list or raising a custom exception to handle this case more gracefully.